frontend/src/container/QueryBuilder/filters/QueryBuilderSearch/style.ts

```diff
@@ -15,3 +15,27 @@ export const TypographyText = styled(Typography.Text)<{
 export const StyledCheckOutlined = styled(CheckOutlined)`
 	float: right;
 `;
+
+export const SelectOptionContainer = styled.div`
```

Comment:
We could have created a component style file and added new styles there. When working on any piece of code, please try and improve it according to the latest standards. This will reduce rework.